sklearn/linear_model/sag.py

```diff
@@ -261,26 +272,42 @@ def sag_solver(X, y, sample_weight=None, loss='log', alpha=1.,
     if max_squared_sum is None:
         max_squared_sum = row_norms(X, squared=True).max()
     step_size = get_auto_step_size(max_squared_sum, alpha_scaled, loss,
```

Comment:
No, it depends on L which is the Lipschitz constant of the gradient of the smooth objective part (i.e logistic + optional l2).